identity/handler.go

```diff
@@ -137,6 +138,13 @@ type listIdentitiesParameters struct {
 	// required: false
 	// in: query
 	CredentialsIdentifier string `json:"credentials_identifier"`
+
+	// IncludeCredentials when set to true includes all of the identity's credentials.
+	//
+	// default: false
+	// required: false
+	// in: query
+	IncludeCredentials bool `json:"include_credentials"`
```

Comment:
It would be nice if this behaved the same way the get identity API works for consistency. There, it's a string slice, see: https://github.com/ory/kratos/blob/1e1f74ee0e5ffccf1aff7d80cdce5d4befb328a8/identity/handler.go#L225